pkg/bundle/bundle.go

```diff
@@ -113,17 +115,32 @@ func (b *ProtobufBundle) validate() error {
 	return nil
 }
 
+// MediaTypeString returns a mediatype string for the specified bundle version.
+// The function returns an error if the resulting string does validate.
+func MediaTypeString(version string) (string, error) {
+	if version == "" {
+		return "", fmt.Errorf("unable to build media type string, no version defined")
+	}
+
+	mtString := fmt.Sprintf("%s+json;version=%s", mediaTypeBase, strings.TrimPrefix(version, "v"))
```

Comment:
This is dependent on version now, since we made a change to be compliant with OCI as per https://github.com/sigstore/protobuf-specs/issues/165.

v0.1 and v0.2 should be `application/vnd.dev.sigstore.bundle+json;version=0.1` and `application/vnd.dev.sigstore.bundle+json;version=0.2` respectively, and for v0.3 onwards, `application/vnd.dev.sigstore.bundle.v0.%s+json` where `%s` is the trimmed version.

Reply:
Ah thanks for the pointer @haydentherapper I was wondering if they were interchangeable. OK, no it will return the old style for 0.1 - 0.2 and the new one for the rest. I've added cases to test them.